src/org/traccar/protocol/WatchProtocolDecoder.java

```diff
@@ -174,41 +182,45 @@ public String getManufacturer() {
         return manufacturer;
     }
 
+    protected int nextIndexOf(ByteBuf buf, char delimiter) {
```

Comment:
Not a big fan of one-line methods. It makes it harder to read the code.

Reply:
duplicate code makes it harder to maintain :) Besides IMHO nextIndexOf is clearer than to look at the original long lines.

Reply:
I don't agree with this. It's one line of code. It's easier to understand code when you know what it's doing and you don't need to refer to a separate method.